CMakeLists.txt

```
@@ -139,6 +145,11 @@ if(USE_CUDA)
   cuda_include_directories(BEFORE "${CMAKE_SOURCE_DIR}/third_party/thrust/dependencies/libcudacxx/include")
 endif(USE_CUDA)
 
+if(USE_ASAN)
+  message(STATUS "Use ASAN")
```

Is ASAN AddressSanitizer?
```suggestion
  message(STATUS "Use AddressSanitizer")
```